### DISCUSSION TITLE
哪位大佬方便告诉我如何更改卡池抽卡角色

### DISCUSSION THREAD

**Discussion by XIAOKU2300** [Melledy/Grasscutter#1999] [Q&A]
我的抽卡池打不开，是不是我有什么东西没有设置成功
，哪位大佬方便告诉我啊，我是个小白，啥都不懂，要是有教程链接的话能不能发出来
谢谢了啊

**Comment by Coldperseverance** [comment]
https://github.com/Zhaokugua/Grasscutter_Banners 我相信你可以在这里找到答案

**Reply by XIAOKU2300** [reply]
非常感谢你的帮助
